### PR TITLE
[lazylibrarian] Move to Common

### LINKED ISSUE #527: [LazyLibrarian] Move to Common Library
LazyLibrarian is a quite simple chart it looks like.
Yet it's not yet using the common chart.

### PR DESCRIPTION
**Pull request by Ornias1993**
**Description of the change**

- This PR moves the LazyLibrarian chart to use the common dependency chart
- Updates the chart to use latest docker container
- and alters documentation accordingly

**Benefits**

- Moving to the common chart should decrease maintenance load on the maintainers

**Possible drawbacks**

- It containers a few breaking changes, which are documented accordingly in the readme

**Applicable issues**

- fixes #527

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/lazylibrarian/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# LazyLibrarian helm chart
+# LazyLibrarian
 
 This is a helm chart for [LazyLibrarian](https://gitlab.com/LazyLibrarian/LazyLibrarian.git) based on the [container image provided by LinuxServer.io](https://hub.docker.com/r/linuxserver/lazylibrarian/).
```
Comment (bjw-s) [inline charts/lazylibrarian/README.md:3]:
```suggestion
This is a helm chart for [LazyLibrarian](https://gitlab.com/LazyLibrarian/LazyLibrarian.git).

**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new/choose)**
```

File: charts/lazylibrarian/values.yaml [hunk 2]
```diff
@@ -1,153 +1,42 @@
...
-  annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: ""
-
-podAnnotations: {}
-
-podSecurityContext: {}
-  # fsGroup: 2000
+  tag: version-047f91af
```
Comment (bjw-s) [inline charts/lazylibrarian/values.yaml:6]: Is this version the same as `appVersion` in `chart.yaml`? May be better to keep them in sync

File: charts/lazylibrarian/values.yaml [hunk 3]
```diff
@@ -1,153 +1,42 @@
...
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+env: {}
+  # TZ: UTC
+  # PUID: 1001
+  # PGID: 1001
+  # DOCKER_MODS=linuxserver/mods:lazylibrarian-ffmpeg
+  # DOCKER_MODS=linuxserver/calibre-web:calibre
```
Comment (bjw-s) [inline charts/lazylibrarian/values.yaml:20]: I don't think you can have the same env var twice, the second one would overwrite the first. Not sure about the lsio syntax to load multiple docker mods though.

**Review by bjw-s — CHANGES_REQUESTED** [review]

**Comment by bjw-s** [comment]
Thank you for taking the time to create this PR! I've left a few small comments, but in general it looks good!

File: charts/lazylibrarian/values.yaml (hunk 2 above)
Comment (Ornias1993) [reply charts/lazylibrarian/values.yaml:6]: App version has not changed, latest release is still 1.7.2

File: charts/lazylibrarian/values.yaml (hunk 3 above)
Comment (Ornias1993) [reply charts/lazylibrarian/values.yaml:20]: Its a comment, just to show users where to put it. 
I don't even know if those work at all, as it isn't part of any testing anyway.

Comment (Ornias1993) [reply charts/lazylibrarian/values.yaml:20]: It looks like in the old code they are just appended after eachother.

Comment (Ornias1993) [reply charts/lazylibrarian/values.yaml:21]: Added a warning comment about using docker mods, to shift blame in case of layer 8 errors.

**Review by bjw-s — APPROVED** [review]

**Pull request merged**